patches/core/ungoogled-chromium/add-licenses-to-credits.patch

```diff
@@ -0,0 +1,853 @@
+--- /dev/null
++++ b/LICENSE.bromite
+@@ -0,0 +1,674 @@
++                    GNU GENERAL PUBLIC LICENSE
++                       Version 3, 29 June 2007
```

Comment:
If I recall, the patches we got from Bromite are under our BSD 3-clause license, otherwise it would be tricky to figure out what parts of the code are under GPL v3 and what other parts are BSD 3-clause (particularly if we consider the patched code instead of the patch files; how does it work for patches that mostly remove code?)

I see two options:
1. We create a BSD 3-clause license that @csagan5 can agree with
2. We remove/rewrite (most of) the patches we incorporated from Bromite. I think we can remove the patches under `extra/bromite`, especially the fingerprinting patches since I'm not sure how useful those fingerprinting mechanisms are (people have reported websites breaking due to them).

Reply:
Is it useful to split this PR into two then? The one with WebRTC-things and the other one fixing licenses?

Reply:
I've commented on https://github.com/bromite/bromite/issues/589 regarding that patch.

Reply:
@qua3k the patch has been tested to work as [intended](https://github.com/bromite/bromite/wiki/WebRTC#bromite-configuration), there are currently no known issues/bugs with it.